src/chromium/crPage.ts

```diff
@@ -215,10 +216,14 @@ export class CRPage implements PageDelegate {
   }
 
   _onAttachedToTarget(event: Protocol.Target.attachedToTargetPayload) {
-    if (event.targetInfo.type !== 'worker')
+    const session = CRConnection.fromSession(this._client).session(event.sessionId)!;
+    if (event.targetInfo.type !== 'worker') {
+      // Ideally, detaching should resume any target, but there is bug in the backend.
+      session.send('Runtime.runIfWaitingForDebugger').catch(debugError);
```

Comment:
Don't we need to call this for worker targets too?

Reply:
We do at the line 235.